src/main/java/edu/ucsb/cs156/example/entities/User.java

```diff
@@ -33,4 +34,16 @@ public class User {
   private String locale;
```

Comment:
Where is the Google information?

Shouldn't we have the Google OAuth userid and email in here somewhere?